style.css

```diff
@@ -608,6 +610,8 @@ form {
 
 .footer li button:hover {
   transform: scale(1.05);
+  background: #2230D2;
+  color: #fff;
```

Comment:
- Good job adding the hover effect to the button :+1: , as suggested by the previous reviewer [here](https://github.com/Tracy-miranja/mobile-version-/pull/3/#discussion_r1072201787), kindly add the active and pressed effect too :recycle: